packages/nocodb/src/lib/services/util.svc.ts

```diff
@@ -56,6 +56,7 @@ export async function appInfo(param: { req: { ncSiteUrl: string } }) {
     ee: Noco.isEE(),
     ncAttachmentFieldSize: NC_ATTACHMENT_FIELD_SIZE,
     ncMaxAttachmentsAllowed: +(process.env.NC_MAX_ATTACHMENTS_ALLOWED || 10),
+    isCloud: !!process.env.NC_CLOUD,
```

Comment:
if users define `NC_CLOUD=false`, `isCloud` would become `true` instead of `false` since `!!'false'` actually returns `true`.